From c6409e552518d91a45ddf5ce96d591dcbfed37f9 Mon Sep 17 00:00:00 2001
From: root <root@ostaphep.cern.ch>
Date: Sat, 22 May 2021 10:23:18 +0200
Subject: [PATCH] fix executables path

---
 ostap/scripts/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ostap/scripts/CMakeLists.txt b/ostap/scripts/CMakeLists.txt
index c43f8c6..5258910 100644
--- a/ostap/scripts/CMakeLists.txt
+++ b/ostap/scripts/CMakeLists.txt
@@ -1,14 +1,14 @@
 
 install(FILES ostap 
-        DESTINATION scripts
+        DESTINATION bin
         PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 
 install(FILES ostap-check-dependencies 
-        DESTINATION scripts  
+        DESTINATION bin  
         PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 
 install(FILES pplaunch
-        DESTINATION scripts
+        DESTINATION bin
         PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 
 configure_file (
-- 
1.8.3.1

