From e5b9d4eab4002b708f723528aca75b5a80bd5815 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Wed, 21 Apr 2021 13:27:55 +0200
Subject: [PATCH] Add missing include

---
 source/include/Ostap/Primitives.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/include/Ostap/Primitives.h b/source/include/Ostap/Primitives.h
index 3d620e5..1456813 100644
--- a/source/include/Ostap/Primitives.h
+++ b/source/include/Ostap/Primitives.h
@@ -6,6 +6,7 @@
 // ============================================================================
 // STD&STL
 // ============================================================================
+#include <algorithm>
 #include <functional>
 #include <cmath>
 // ============================================================================
-- 
2.27.0

