--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -1,21 +1,21 @@

 install ( FILES       ostap
-          DESTINATION scripts
+          DESTINATION bin
           COMPONENT   scripts
           PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)

 install ( FILES       check-dependencies
-          DESTINATION scripts
+          DESTINATION bin
           COMPONENT   scripts
           PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)

 install ( FILES       sync-dirs
-          DESTINATION scripts
+          DESTINATION bin
           COMPONENT   scripts
           PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)

 install ( FILES       pplaunch
-          DESTINATION scripts
+          DESTINATION bin
           COMPONENT   scripts
           PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)

